app/assets/javascripts/components/sidebar_footer.es6.jsx

```diff
@@ -4,31 +4,34 @@ class SidebarFooter extends Component {
     return {
       container: {
         display: 'flex',
+        alignItems: 'center',
         position: 'relative',
-        padding: '20px',
+        padding: '14px',
+        marginTop: '14px',
         marginLeft: '14px',
-        alignItems: 'center',
-        justifyContent: 'center',
       },
-      text: {
-        paddingRight: '5px',
+      label: {
+        position: 'relative',
+        fontSize: '14px',
+        paddingRight: '8px',
       },
       image: {
-       height: '100%',
-       width: '100%',
+        position: 'relative',
+        width: '18px',
+        height: '18px',
       },
     };
   }
 
   get clickableStyles() {
-    return { 
-      default: { 
-        maxHeight: '34px',
-        maxWidth: '34px',
-        transition: 'transform 0.5s ease-out',
+    return {
+      default: {
+        lineHeight: '0px',
```

Comment:
@sonia-y do you remember how when the gear was spinning it seemed a little off center? For some reason adding `lineHeight: '0px'` fixed that. I'm not entirely sure why but intuitively I just guessed it and it worked.